R/qs_metabolic.R

```diff
@@ -3,12 +3,12 @@
 # Manual edits are not recommended, as changes may be overwritten.
 #' Questionnaire Dataset-metabolic
 #'
-#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies
+#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {admiralmetabolic} as well as the ADCOEQ code are for not-for-profit use only within {admiralmetabolic} and pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
```

Comment:
```suggestion
#' A SDTM QS dataset containing COEQ data (Control of Eating Questionnaire) for metabolic studies. Note that University of Leeds are the copyright holders of the Control of Eating Questionnaire (CoEQ) and the test data included within {pharmaversesdtm} as well as the ADCOEQ code are for not-for-profit use only within pharmaverse-related examples/documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
```

I cant quite remember, but I think this needs to be changed in [inst/extdata/sdtms-specs.json](https://github.com/pharmaverse/pharmaversesdtm/pull/144/files#diff-5db1936bda11ada4ca8d0a7083f7e554fd054c553858068d1884c60c11ba9f90)?

Reply:
@AndersAskeland ah yes good spot. I confirm that it needs to be changed not in `qs_metabolic.R` but `inst/extdata/sdtms-specs.json`. I've left a suggestion below @Siddhesh2097 but you'll have to regenerate the documentation. Text was approved but again none of the substance is changing, just the location of the data within the wider pharmaverse. I think it's fine, but equally @AndersAskeland if you'd like to give a courtesy email to UoLeeds letting them know of the move, maybe that'll cover us on all bases.

Reply:
@AndersAskeland thank you for noticing this. It's updated now
@manciniedoardo

Reply:
@AndersAskeland let us know how you want to proceed!